src/responder/secrets/providers.c

```diff
@@ -42,10 +50,43 @@ static int sec_map_url_to_user_path(struct sec_req_ctx *secreq,
         return ENOMEM;
     }
 
-    DEBUG(SSSDBG_TRACE_LIBS, "User-specific path is [%s]\n", *mapped_path);
+    DEBUG(SSSDBG_TRACE_LIBS,
+          "User-specific secrets path is [%s]\n", *mapped_path);
     return EOK;
 }
 
+static int kcm_map_url_to_user_path(struct sec_req_ctx *secreq,
```

Comment:
Can you change the function name to kcm_map_url_to_path ? We are not mapping to any "user" path in here.